applications/endoscopy_tool_tracking_distributed/cpp/main.cpp

```diff
@@ -0,0 +1,221 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights
+ * reserved. SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <getopt.h>
+#include <holoscan/operators/format_converter/format_converter.hpp>
+#include <holoscan/operators/holoviz/holoviz.hpp>
+#include <holoscan/operators/video_stream_replayer/video_stream_replayer.hpp>
+#include <iostream>
+#include <lstm_tensor_rt_inference.hpp>
+#include <string>
+#include <tool_tracking_postprocessor.hpp>
+#include "holoscan/holoscan.hpp"
+
+using namespace holoscan;
+class VideInputFragment : public holoscan::Fragment {
```

Comment:
`Vide` -> `Video`, here and throughout

```suggestion
class VideoInputFragment : public holoscan::Fragment {
```